api/swagger-spec/v1beta1.json

```diff
@@ -103,18 +135,50 @@
     "description": "API at /api/v1beta1 version v1beta1",
     "operations": [
      {
-      "type": "v1beta1.EndpointsList",
+      "type": "*json.watchEvent",
```

Comment:
> On Mar 30, 2015, at 8:17 PM, Brian Grant <notifications@github.com> wrote:
> 
> In api/swagger-spec/v1beta1.json:
> 
> > @@ -103,18 +135,50 @@
> >      "description": "API at /api/v1beta1 version v1beta1",
> >      "operations": [
> >       {
> > -      "type": "v1beta1.EndpointsList",
> > +      "type": "*json.watchEvent",
> Looks like fallout from @smarterclayton's watch changes.
> 
> watchEvent looks like a backdoor unversioned API type (from watch/json/types.go) that's not supposed to be public
It's versioned, but we could not run conversion at the time it was created because of limitations in the underlying conversion method.  It is the real object that you get when you watch.  Probably can just create a type alias.

> I'm not sure the * syntax works -- it may well break swagger validation, which currently passes
Type aliasing should clear that up.
> We've lost documentation of the watch payload type
Technically... there are multiple watch payload types.  Status and the local Kind.  I'd err on the side of describing both.
> —
> Reply to this email directly or view it on GitHub.
>